src/components/Redemptions/RedemptionsPage.tsx

```diff
@@ -40,7 +40,43 @@ const mapDispatchToProps = {
   showNotification,
 };
 
-type IProps = IStateProps & IDispatchProps & ISubscriptionProps<[IDAOState[], any[]]>
+type IProps = IStateProps & IDispatchProps & ISubscriptionProps<any[]>
```

Comment:
Possibly something better than the `any` in `ISubscriptionProps<any[]>`?  If not, document it?

Reply:
this is the way things are with special queries

Reply:
I don't understand, it is an array of objects that implement an interface that is just begging to be defined.